source/Nuke.Common/Tools/ReportGenerator/ReportGeneratorTasks.cs

```diff
@@ -1,11 +1,25 @@
-// Copyright 2019 Maintainers of NUKE.
+﻿// Copyright 2019 Maintainers of NUKE.
 // Distributed under the MIT License.
 // https://github.com/nuke-build/nuke/blob/master/LICENSE
 
+using JetBrains.Annotations;
 using Nuke.Common.Tooling;
 
 namespace Nuke.Common.Tools.ReportGenerator
 {
+    [PublicAPI]
+    public class RaportGeneratorVerbosityMappingAttribute : VerbosityMappingAttribute
```

Comment:
Shouldn’t it be ‚Report‘ instead of ‚Raport‘?

Reply:
Yes, you're correct. Nice catch.
- [ ] fix a typo